content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,289 @@
+---
+linkTitle: Custom Lighting Tutorial with Flipbook Animation
+title: Custom Lighting Tutorial with Flipbook Animation
+description: A tutorial for custom lighting using a flipbook animation with six-point lighting in the Atom renderer of the Open 3D Engine (O3DE).
+toc: true
+---
+In this tutorial, we cover how to make your own material type that uses a custom surface with custom lighting. In this case, we will be using a flipbook animation with 6-point lighting. This material type is especially useful for use on effects such as smoke and clouds, since we want to animate movement while still capturing correct lighting. However, traditional lighting methods would not work because it is a 2D texture. Therefore, to apply the correct lighting, we will need a custom material type that uses custom lighting. We follow [this technique](https://realtimevfx.com/t/smoke-lighting-and-texture-re-usability-in-skull-bones/5339) to use six points (top, bottom, left, right, front, and back) and determine which points of the six should be used to compute a lightmap. 
```

Comment:
We have "6-point", "six-point", and "six point" in various places. Let's use "six-point" everywhere.

Also, let's change "to use six points (top, bottom, left, right, front, and back) and determine which points of the six should be used to compute a lightmap."

to be
"to use six tangent lightmaps (top, bottom, left, right, front, and back) to approximate how a texel should be lit from any given direction."